contrib/stack/topsStack/stackSentinel.py

```diff
@@ -166,8 +166,11 @@ def createParser():
     parser.add_argument('-useGPU', '--useGPU', dest='useGPU',action='store_true', default=False,
                         help='Allow App to use GPU when available')
 
-    parser.add_argument('--num-proc', '--num-process', dest='numProcess', type=int, default=1,
-                        help='number of parallel processes (where applicable) (default: %(default)s).')
+    parser.add_argument('--num_proc', '--num_process', dest='numProcess', type=int, default=1,
+                        help='number of tasks running in parallel in each run file (default: %(default)s).')
+
+    parser.add_argument('--num_proc4topo', '--num_process4topo', dest='numProcess4topo', type=int, default=1,
+                        help='number of parallel processes (for topo only) (default: %(default)s).')
```

Comment:
Not a big deal at all, but it would be more convenient for non-HPC users if we could tie `numProcess4topo` to `numProcess` in `cmdLineParse()` if the later is specified in command line while the former is not, such as below:

```python
    if any(i in iargs for i in ['--num_proc', '--num_process']) and all(i not in iargs for i in ['--num_proc4topo', '--num_process4topo']):
        inps.numProcess4topo = inps.numProcess
```

Reply:
@yunjunz  please check if it is working fine as you suggested